docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -105,50 +101,79 @@ Mapping examples:
 [EDGEX_STARTUP_DURATION]: ../../microservices/configuration/CommonEnvironmentVariables/#edgex_startup_duration
 [ADD_SECRETSTORE_TOKENS]: ../../security/Ch-Configuring-Add-On-Services/#configure-the-services-secret-store-to-use
 
-!!! note
-    The config options are supported as of EdgeX 2.2 and are disabled by default!
-
-    Setting `app-options=true` is necessary to enable their support.
-
-    Enabling app options will migrate internal env options and persistently remove any newly set env option.
-
-For example, to change the service port of the core-data service on `edgexfoundry` snap to 8080:
-```bash
-snap set edgexfoundry app-options=true
-snap set edgexfoundry apps.core-data.config.service-port=8080
-```
+!!! Example 
+    To change the service port of the `core-data` service on `edgexfoundry` snap to 8080:
+    ```bash
+    snap set edgexfoundry apps.core-data.config.service-port=8080
+    ```
+    This would internally export `SERVICE_PORT=8080` to `core-data` service.
 
-The services load the set config options on startup. If the service has already started, a restart is necessary to load them.
+!!! Note
+    The services load the set config options on startup. If a service has already started, a restart will be necessary to load the configurations.
 
-#### Common configuration examples
+#### Examples
 ##### Disabling security
 [disabling security]: #disabling-security
-
+    
 !!! Warning
     Disabling security is NOT recommended, unless for demonstration purposes, or when there are other means to secure the services.
 
-    The snap will NOT allow the Secret Store to be re-enabled. The only way to re-enable the Secret Store is to re-install the snap.
-    
-The Secret Store is used by EdgeX for secret management (e.g. certificates, keys, passwords). Use of the Secret Store by all services can be disabled globally. Note that doing so will also disable the API Gateway, as it depends on the Secret Store.
+    The [platform snap] snap does NOT allow the security to be re-enabled. The only way to re-enable it is to re-install the snap.
+
+Disabling security involves two steps:
 
-The following command disables the Secret Store and in turn the API Gateway:
+1. Stopping the security services and disabling them so that they don't run again.
+2. Configuring EdgeX services to NOT use the Secret Store by setting [EDGEX_SECURITY_SECRET_STORE](../../microservices/configuration/CommonEnvironmentVariables/#edgex_security_secret_store) to false.
```

Comment:
The `EdgeX services` is confusing. what services does it indicate? platform snap, device/app snap, or both?

[Line 325-336](https://github.com/edgexfoundry/edgex-docs/blob/49caa2a569866226575e857bccb47c78c52e794c/docs_src/getting-started/Ch-GettingStartedSnapUsers.md#platform-snap) only mentioned platform snap's services are EdgeX services, but I guess `Configuring EdgeX services` means device/app/ekuiper/ui services here.

Reply:
Do you think a terminology section is necessary?

Reply:
All first-party EdgeX services. So not Kong, Vault, Consul, Redis, ...

See also https://github.com/edgexfoundry/edgex-docs/pull/939#discussion_r1082441244

Reply:
Added a sentence to clarify.